File: app/src/main/java/com/widiarifki/screentest/presentation/guest/IGuestView.java
package com.widiarifki.screentest.presentation.guest;

import com.widiarifki.screentest.model.Guest;

import java.util.List;

/**
 * Created by widiarifki on 27/02/2018.
 */

public interface IGuestView {
    void showList(List<Guest> guestList);
    void showGetDataFail();
    void showProgressDialogue(boolean isShowing);
    void showDialogue(String title, String info);
}

File: app/src/main/java/com/widiarifki/screentest/presentation/guest/IGuestView.kt
package com.widiarifki.screentest.presentation.guest

import com.widiarifki.screentest.model.Guest

/**
 * Created by widiarifki on 27/02/2018.
 */

interface IGuestView {
    fun showList(guestList: List<Guest>)
    fun showDialogMessage(title: String?, info: String)
}
